### ISSUE TITLE
try/catch blocks empty?

### ISSUE THREAD

**Issue by johnd0e** [IITC-CE/ingress-intel-total-conversion#58]
In many places of code with inline include (@@INCLUDERAW...@@) I see patterns like this:

```
try { console.log('Loading included JS now'); } catch(e) {}
```

**Comment by modos189** [comment]
They don't seem empty, it was supposed to process `console.log()` in `try{}`
I don't know why.

**Comment by johnd0e** [comment]
I suppose the initial intention could be to put whole included file into `try` block.

**Comment by johnd0e** [comment]
We can resolve this like here: #141.

Or maybe we do not need such error handling at all.
IMHO we can just remove such empty try/catch blocks.

**Edit**
I think we should do like this: 6d3aa5641cfb480486951dbc56079bc3518ac8e0

**Comment by johnd0e** [comment]
Fixed in #134
